sdks/python/apache_beam/yaml/standard_io.yaml

```diff
@@ -0,0 +1,53 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This file enumerates the various IOs that are available by default as
+# top-level transforms in Beam's YAML.
+#
+# Note that there may be redundant implementations. In these cases the specs
+# should be kept in sync.
+# TODO(yaml): See if this can be enforced programmatically.
+
+- type: renaming
+  transforms:
+    'ReadFromBigQuery': 'ReadFromBigQuery'
+    'WriteToBigQuery': 'WriteToBigQuery'
+  config:
+    mappings:
+      'ReadFromBigQuery':
+        query: 'query'
+        table: 'tableSpec'
+        fields: 'selectedFields'
+        row_restriction: 'rowRestriction'
+      'WriteToBigQuery':
+        table: 'table'
+        create_disposition: 'createDisposition'
+        write_disposition: 'writeDisposition'
+        error_handling: 'errorHandling'
+    underlying_provider:
+      type: beamJar
+      transforms:
+        'ReadFromBigQuery': 'beam:schematransform:org.apache.beam:bigquery_storage_read:v1'
+        'WriteToBigQuery': 'beam:schematransform:org.apache.beam:bigquery_storage_write:v1'
```

Comment:
Should this be `v2`?

Reply:
Oh, yes. (I changed this at the last minute just to be safe...)